dns/main.go

```diff
@@ -29,6 +29,10 @@ func main() {
 	// PLEASE use 9121! hash_server.go assumes this port for forwarding
 	httpAPIPort := flag.Int("port", 9121, "dns HTTP API server port")
 	join := flag.Bool("join", false, "join an existing cluster")
+	migrate := flag.Bool("migrate", false, "need to coordinate migration")
+	// include the port in hash server ip addr
+	hashServer := flag.String("hash", "", "the ip address of a hash server to help with migration")
+	config := flag.String("config", "", "the path to the json file containing the config for this cluster")
```

Comment:
Add a comment to explain the format of the config (mostly how it is different from hash_server config)